packages/eslint-config/base.cjs

```diff
@@ -202,13 +202,40 @@ module.exports = {
         'unicorn/prefer-module': 'off',
       },
     },
-    // Relax rules that don't make sense for testing
+    // Rules for tests
     {
-      files: ['**/__tests__/**', '**/*.spec.ts'],
+      files: ['**/__tests__/**', '**/*.test.ts', '**/*.spec.ts'],
```

Comment:
Since we are enforcing tests with the `.spec.ts` extension, can we drop `**/*.test.ts`?

Also, is there a good way to enforce that test files are in an `__tests__` dir?

Reply:
> Since we are enforcing tests with the .spec.ts extension, can we drop **/*.test.ts?

Without `.test.ts` in there, the files won't be linted, and therefore the "enforce `.spec.ts`" rule will not trigger 🐔🥚. Also we have older projects that haven't switched to the newer convention, and I want to make sure they can still benefit

> Also, is there a good way to enforce that test files are in an __tests__ dir?

I messed around with the filename format rule, and I _think_ it's possible. But the main barrier to this is that non-unit tests (e.g. E2E tests, or the package.json tests in this repository) aren't and shouldn't be inside a `__tests__` dir, so the waters are a little muddy

Reply:
Sounds good! I trust your judgement!